cpp/modules/deck.gl/core/src/shaderlib/project/viewport-uniforms.cpp

```diff
@@ -162,49 +166,52 @@ auto calculateViewportUniforms(const std::shared_ptr<Viewport>& viewport, double
   auto distanceScales = viewport->getDistanceScales();
 
   auto viewportSize = Vector2<double>(viewport->width, viewport->height) * devicePixelRatio;
+  auto antimeridian = (viewport->isGeospatial ? viewport->longitude : 0.0) - 180.0;
 
+  // TODO(ilija@unfolded.ai): Aren't designated initializers part of C++20?
+  // https://en.cppreference.com/w/cpp/language/aggregate_initialization
+  // TODO(ilija@unfolded.ai): uWrapLongitude and uModelMatrix not set, set them to default values explicitly if they
```

Comment:
- uWrapLongitude is just a boolean should get a default value from the base layer props (default should match JS).